killmyjobs

```diff
@@ -39,12 +39,11 @@ fi
 if [ "${jobids}" = "" ]; then
     echo "well I would...but you've got no jobs to kill m8"
 else
-    echo "${exclude[@]}"
     if [[ -n ${exclude+1} ]]; then
-	echo "excluding ${exclude[@]}" 
```

Comment:
Does removing the quotation marks change the format of the output? It doesn't seem to make a difference to me.

(If there's no difference, I would prefer to have quotation marks around the message to be printed, but that's just a style preference so I wouldn't insist on it being changed.)

Reply:
Yeah I think you're right and, because it's an array, you don't need to remove the quotes to make it not have newlines. I needed it in the echo below to remove newlines. I'll put them back just for you :smile_cat: